README.md

```diff
@@ -31,15 +37,20 @@ Gulp concats this with materilize and minifies it to `dist/js`
 ## Development
 
 ### Local Development
+
 * When running the files locally, run it using a web server or you will have problems with Same-origin policy. _no longer true as site is regenerated_
+
 A simple solution is to run `gulp dev` which will compile the less, watch it and start a webserver at techweek.dev:8000
 
 ### Docker
+
 ```
 docker-compose up install
 docker-compose up dev
 ```
-### CSS
-All the css is compiled from less in the less directory.
+
+### sCSS
```

Comment:
`There are two syntaxes available for Sass. The first, known as SCSS (Sassy CSS)`

Twould appear to be `SCSS`